release-content/0.15/migration-guides/15591_Text_rework.md

```diff
@@ -1,8 +1,28 @@
-TODO: very breaking
+The `Text` API in Bevy has been overhauled in several ways as part of Bevy 0.15.
+There are several major changes to consider:
 
-__Accessing text spans by index__
+- `ab_glyph` has been replaced with `cosmic-text`. These changes are mostly internal and the majority of users will not interact with either text backend directly.
+- each text section is now stored as a distinct entity within the standard hierarchy, rather than as a `Vec<TextSection>` on the `Text` component. Children of `Text`/`Text2d` entities with `Text` `Text2d` components will act as additional text sections.
```

Comment:
Perhaps not a migration issue, but somewhere I think it useful to mention that since text sections are entities, you can now insert components on them, which is useful for things like:
* Dynamically-computed text
* Picking behavior
* Animated text styles.

Reply:
Yeah, this belongs in the release notes, not the migration guide.